From b5601cff08accb45befdd523b35337c96550c547 Mon Sep 17 00:00:00 2001
From: He Sheng <hesheng@wxiat.com>
Date: Tue, 10 May 2022 14:21:21 +0800
Subject: [PATCH] sw64: remove unused IO_CONCAT
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: b5601cff08accb45befdd523b35337c96550c547
Modified-by-SEL: No


Sunway inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I56OLG

--------------------------------

IO interface of chipset is wrapped as a hook in sw64_platform_ops.

Signed-off-by: He Sheng <hesheng@wxiat.com>

Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/include/asm/io.h | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/arch/sw_64/include/asm/io.h b/arch/sw_64/include/asm/io.h
index e8e80c761467..3138665133e1 100644
--- a/arch/sw_64/include/asm/io.h
+++ b/arch/sw_64/include/asm/io.h
@@ -64,13 +64,6 @@ static inline void * __deprecated bus_to_virt(unsigned long address)
 }
 #define isa_bus_to_virt bus_to_virt
 
-/*
- * There are different chipsets to interface the sw64 CPUs to the world.
- */
-
-#define IO_CONCAT(a, b)		_IO_CONCAT(a, b)
-#define _IO_CONCAT(a, b)	a ## _ ## b
-
 #include <asm/sw64io.h>
 
 /*
-- 
2.33.0

